Sources/Sentry/SentrySDK.m

```diff
@@ -464,9 +464,19 @@ + (void)installIntegrations
     NSMutableArray<NSString *> *integrationNames =
         [SentrySDK.currentHub getClient].options.integrations.mutableCopy;
 
+    NSArray<Class> *defaultIntegrations = SentryOptions.defaultIntegrationClasses;
```

Comment:
`m`: It would be good to add a comment somewhere on why we interact with the classes directly and require this dictionary. That isn't obvious.